content/doc/administrate/activity.md

```diff
@@ -0,0 +1,38 @@
+---
+type: docs
+title: Activity
+description: Applications activity lifecycle explained
+tags:
+- lifecycle
+keywords:
+- activity
+- unreachable
+- timeout
+- diskfull
+- scaling
+- maintenance
+type: docs
+---
+
+Follow your application activities and lifecycle from the **Activity** option in the application menu. Every time your application restarts, a new block appears on the timeline. Here's what you might see on the timeline:
```

Comment:
It can be a bit confusing.
Redeploy can be a restart or a rebuild
Dunno if this distinction is relevant :thinking: 
It is true that a block appear if you restart or rebuild
But you didn't mention here other source of apparition (maintenance / monitoring action described on the second half) or the possibility they exist, or the fact that these block likely come from user activity...
But maybe it is better to introduce block and after introduce non-user cause

Reply:
Another kind of block exist : UNDEPLOY, when you stop your application
![image](https://github.com/user-attachments/assets/11572026-e0e0-4743-9e29-1fa19dac2197)
In that case there is no commit id, nor mean to redeploy this commit

Reply:
```suggestion
Follow your application activities and lifecycle from the **Activity** option in the application menu. Every time your application restarts or rebuilds, a new block appears on the timeline. Here's what you might see on the timeline:
```